igrica.js

```diff
@@ -0,0 +1,62 @@
+let okvir = document.getElementById('okvir');
+let plava_kocka = document.getElementById('kocka');
+let zelena_kocka = document.getElementById('kocka-zelena');
+let crvena_kocka = document.getElementById('kocka-crvena');
+
```

Comment:
ovi let-ovi bi trebali da budu constovi, jel si provukao kroz linter?

Reply:
Nema lintera, nisam uradio npm i. Ovo je vise radi vezbe, a ne ozbiljan projekat.

Reply:
Moze lako da preraste u ozbiljniji projekat. Ima dobru osnovu svakako

Reply:
Probaj da se navikneš na linter što pre.

Reply:
Ovo je projekat samo za vezbu pa zato nisam koristio linter. Inace cu svakako da ga koristim